src/client.js

```diff
@@ -43,6 +43,26 @@ export function streamEvents(system, callback) {
     });
 }
 
+export function pingger(system) {
```

Comment:
This is a  bit "meh". If it really needs to be pinged, let's at least fix the spelling to `pinger` :)

I checked our `cockpit.http` code in cockpit.js, and it doesn't expose either its internal `channel` nor the "close" event handler. However, it may be more elegant to just create a raw stream channel yourself, like

    cockpit.channel({"payload": "stream", "unix": addressPath}).addEventListener("close", () => console.warn("podman died"))

Reply:
That is a good idea to just create a raw channel and listen on that. That should give us a good idea when things go bad in the backend.

Reply:
This has been implemented.